engine-src/Game/LambdaHack/Common/ReqFailure.hs

```diff
@@ -141,6 +147,9 @@ showReqFailure reqFailure = case reqFailure of
   AlterBlockActor -> "blocked by an actor"
   AlterBlockItem -> "jammed by an item"
   AlterNothing -> "wasting time on altering nothing"
+  CloseDistant -> "trying to close a dinstant object"
+  CloseClosed -> "already closed"
+  CloseNothing -> "wasting time on closing nothing"
```

Comment:
I'd change that to something like "no adjacent terrain can be closed". (Probably the `AlterNothing` is similarly vague, but at least then we have to convey a complex thing: not secret tile, not modifiable tile and no embedded items to activate (however, improve it, if you can). And here it's really simple, so we can do better.